src/components/styles/AddBook.css

```diff
@@ -0,0 +1,41 @@
+.add-book-form {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  width: 64em;
+}
+
+.title-add-book {
+  width: 10.875rem;
+  height: 1.5rem;
+  font-family: var(--font-pri) !important;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
```

Comment:
- [ ] Please try as much as possible to ensure that you stick to the Original design as suggested by the previous reviewer. I suggest that you consider removing the height attached. In addition, kindly consider aligning the form inputs and the title to the same level as requested by the previous reviewer ☺️. This will improve the UI of your application 👍.
- [ ] [OPTIONAL] Please try your best to avoid using `!important` in your CSS.

Your design 😎 
![image](https://github.com/iamsjunaid/Bookstore/assets/99394374/73e4d347-968e-470d-81b0-fcdc2c173364)

Zeplin design 😎 
![247830368-250a5e08-386f-4976-b9ed-5e1b67d852ef](https://github.com/iamsjunaid/Bookstore/assets/99394374/df41a7f2-1ebd-47b8-8d8c-95226e5878a5)

Reply:
Thanks for the feedback @AlphaNtihinduka. removed the `!important` and aligned the title `ADD NEW BOOk`.